third_party/microprofile/microprofileui.h

```diff
@@ -2948,7 +2948,8 @@ void MicroProfileDrawMenu(uint32_t nWidth, uint32_t nHeight)
 		float fMs = fToMs * (S.nFlipTicks);
 		float fAverageMs = fToMs * (S.nFlipAggregateDisplay / nAggregateFrames);
 		float fMaxMs = fToMs * S.nFlipMaxDisplay;
-		int nLen = snprintf(FrameTimeMessage, sizeof(FrameTimeMessage)-1, "Time[%6.2f] Avg[%6.2f] Max[%6.2f]", fMs, fAverageMs, fMaxMs);
+		float fps = 1000 / fMs; // maybe fAverageMs is better cuz it's averages?
```

Comment:
- If floating-point numbers are used, it's better to clearly display that in the literals used in the code (`1000.0f` instead of the implicitly converted integer `1000`).
- "maybe" should be a "// TODO(illusion0001): ", but if `fAverageMs` just works now, there's no need to leave technical debt.
- Comments in Xenia code are generally placed on a separate line (but if on the same line, separated with 2 spaces).
- "maybe" → "Maybe", "cuz" → "because", "averages" → "averaged".

Overall though this feature should preferably be implemented in @zeux's upstream for easier updating in the future, possibly with a nicer division by zero handling (though displaying something like 1.#INF is possibly acceptable too).

Reply:
> "cuz" → "because"

Code consistency is one thing, but why are you policing their spelling?

Reply:
English language consistency 🙃

Reply:
I did some testing, it's better to use `fAverageMs` here because converting from `fMs` every frame can produce incorrect results. Because this behavior does not average out the time it takes to draw a frame, one frame can be draw for 8.3ms, the other can spike up to 33, thus a sudden jump in "averaged" frame counting.

Reply:
> If floating-point numbers are used, it's better to clearly display that in the literals used in the code (1000.0f instead of the implicitly converted integer 1000).

Done.

Reply:
Though I should possibly account for if frame time is < 0 because during xenia startup, it does not count frames and leads to NaN.

Reply:
If it could show up during startup, it would probably just show a garbage number for NaN. But it can't because the screen it is used on has to be invoked after startup.